sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtilTest.java

```diff
@@ -502,7 +502,9 @@ public void testGetSizeBytesWhenFileNotFoundBatchRetry() throws Exception {
 
     // 429: Too many requests, then 200: OK.
     when(mockResponse.getStatusCode()).thenReturn(429, 200);
-    when(mockResponse.getContent()).thenReturn(toStream("error"), toStream(content));
+    // Google-http-client:1.39.1 and above does not read content stream of the error response.
```

Comment:
This should not be true unless disconnect is being called somewhere.

Reply:
Thanks. I'll create a test case for google-http-java-client to verify this.

Reply:
It turned out that it was mock setup that had wrong assumption that getStatusCode is called once per response.